docs/api/build_proto.md

```diff
@@ -72,14 +72,14 @@ There are many tools for building proto in Rust, we use [tonic](https://github.c
 1.  Create project
 
     ```bash
-    $ cargo new --lib vald-grpc
```

Comment:
I think it is better to leave `$` here for easy understanding on if it is command or not.

Reply:
It is for user can run command only Copy and Paste